docs/getting-started/new-music-help-faq.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: default
+title: New Music changes in 9.0
+---
+
+# New Music browsing changes in 9.0
+Release 9.0 introduced changes to browsing your music by __New Music__ added to your Lryion music library and offers and additional new browse menu option for viewing music that was most __Recently Updated__.  This guide is intended to provide clarification and further details about this change and answer some frequently asked questions.
+
+## What's changed in 9.0
+### New Music browsing
+Selecting browse by __New Music__ will now display a list of albums sorted by the actual date that the album was first scanned into your Lyrion library. Technically, it is sorting on the oldest date added from the list tracks for a given album as stored in your persist.db (located in your LMS Prefs directory). Since the persist.db survives a "Clear Library and Rescan All" it is able to maintain a correct history of when music was added to your LMS library based on the timestamp it was first scanned. If for some reason you start with a new persist.db, you will lose the history of when you music was added to Lyrion which is why you should consider this an important file to back up just like you maintain good backups of all of your music files.
+
+!!! note
+        As of the latest 9.0.1 release, the first library scan for a new persist.db will populate the Date Added field for each track with the file modified timestamp. All subsequent scans will populate the Date Added with timestamp the file is being scanned. This helps new users or those who must start with a fresh persist.db to start out with a best guess at the correct sort for New Music based on file modified timestamps.
+
+In 9.0 you can also see the Date Added from persist.db in the UI from the context menu by selecting More > More Info.  You will see both Date Added (from persist.db) and Date Modified (from the file modified timestamp as stored in library.db).
+
+### Recently Update Albums browsing
+There is an additional browsing option for __Recently Updated Albums__ in 9.0 that will display a list of albums sorted by the file modified timestamp. This is a more accurate description for how the New Music browsing behaved prior to release 9.0. If you prefer to see a list of albums sorted by file modified date like the New Music was previous to 9.0, then you should use __Recently Updated Albums__ browsing menu.
+
+You can add this menu for each player via __Settings > Player > Additional Browse Modes__.
+
+!!! note
+        If you do not see __Additional Browse Modes__ as an option then you need to install the Additional Browse Modes plugin via the __Plugin Manager__ under Settings.
+
+![Add Browse Mode](assets/new-music/add-browse-modes.png){ align=left }
```

Comment:
`align=left` doesn't work well here, as the text is wrapped on the right. This doesn't work well on smaller screens (eg. the laptop I'm working on right now). I personally prefer to centre such images - which unfortunately is surprisingly complicated to do in markdown...

```
<figure markdown="span">
  ![Add Browse Mode](assets/new-music/add-browse-modes.png)
</figure>
```